src/i_video.c

```diff
@@ -635,7 +635,7 @@ void I_FinishUpdate(void)
         need_resize = false;
     }
 
-    if (uncapped && fpslimit >= TICRATE)
+    if (!uncapped || fpslimit >= TICRATE)
```

Comment:
Should we write just `if (fpslimit >= TICRATE)` there? SDL has it's own fps limiter: https://github.com/libsdl-org/SDL/blob/SDL2/src/render/SDL_render.c#L4214 Maybe we should disable our limiter when vsync is enabled and `fpslimit == native_refresh_rate`.

Reply:
I tried a better solution here: https://github.com/fabiangreffrath/woof/pull/1408/commits/e28677efe6367443bf03947ad3a97d9dd5f4f7e5

> SDL has it's own fps limiter: https://github.com/libsdl-org/SDL/blob/SDL2/src/render/SDL_render.c#L4214 Maybe we should disable our limiter when vsync is enabled and `fpslimit == native_refresh_rate`.

Good idea, I added something similar to that.